src/AccessibilityInsights.Extensions.AzureDevOps/ConfigurationControl.xaml.cs

```diff
@@ -5,6 +5,7 @@
 using AccessibilityInsights.Extensions.Helpers;
 using AccessibilityInsights.Extensions.Interfaces.IssueReporting;
 using Microsoft.VisualStudio.Services.Common;
+using SHDocVw;
```

Comment:
not sure if this came in with the event listener, will double check if its actually needed